src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "react";
+import "./Modal.css";
+import ImageUploading, { ImageListType } from "react-images-uploading";
+import { Post } from "../App/App";
+// import { idText } from "typescript";
+
+interface ModalProps {
+  submitPost: (newPost: Post) => void;
+  closeModal: (event: any) => void;
+}
+
+export const Modal: React.FC<ModalProps> = ({ submitPost, closeModal }) => {
+  const [images, setImages] = useState([]);
+  const [postTitle, setTitle] = useState<string>("");
+  const [postDescription, setDescription] = useState<string>("");
+  const [imageURLS, setImageURLS] = useState<string[]>([]);
+
+  const maxNumber = 2;
+  const onChange = (
+    imageList: ImageListType,
+    addUpdateIndex: number[] | undefined
+  ) => {
+    setImages(imageList as never[]);
+    let onlyURLs = imageList.map((image) => image.data_url);
+    setImageURLS(onlyURLs);
+  };
+
+  const clearState = () => {
+    setImages([]);
+    setTitle("");
+    setDescription("");
+  };
+
+  const addPost = (event: React.MouseEvent<HTMLElement>) => {
+    event.preventDefault();
+    const newPost = {
+      id: 2,
+      title: postTitle,
+      description: postDescription,
+      image: imageURLS,
+      user: { id: 4, username: "workerBee1" },
+      upvotes: 0,
+      downvotes: 0,
```

Comment:
You can also just omit this info, as far as we're concerned -- we don't need you to send 0s as this is an optional field in our DB

Reply:
Yes, I agree with Molly. We have default values for data like this. Just for future reference

Reply:
Yeah yeah! We just needed to have all the types included so that TS would be happy since we're using mock data, this will be refactored when we move to graphQL